tests/robot-tests/README.md

```diff
@@ -59,137 +58,116 @@ If you intend to run the tests from your local machine, you will also need to cr
-
-
-## Webdriver
-The `webdriver` directory holds `chromedriver`, used by selenium to interact with the browser. If `chromedriver` isn't present in this directory, it is automatically downloaded when the tests are run. You can explicitly download the `chromedriver` version of your choice with `--chromedriver <version>`. Alternatively, you can manually place the `chromedriver` of your choice into the `webdriver` directory. 
+This script is responsible for running the UI tests. All available options for `run_tests.py` can be seen by running `pipenv run python run_tests.py --help`
 
-NOTE: The `run_tests.py` script only downloads `chromedriver` if it doesn't already exist. If you wish the run script to download a different version, you'll first have to delete `chromedriver` from the `webdriver` directory.
+Here is an example:
 
-If you need to change the chromedriver version used by the CI pipeline, it can be done in `scripts/pipeline-run-rf-tests.sh`. You can check [this repository](https://github.com/actions/virtual-environments/tree/master/images) for the version of chrome used on the Azure agent you're using. At the time of writing, the robot tests use the [Ubuntu 1604 image](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1604-README.md)
+- `pipenv run python run_tests.py -f tests/admin/bau/ -i robot -b chrome -e dev`
 
+## **Running tests on the pipeline**
 
-## How do I backup and restore the test data on my local environment?
+After changing the tests and ensuring they pass locally, it is then recommended that you also run your changes against the Dev environment. This can be done by either running the tests locally against dev (`run_tests.py -e dev`) or by utilising the Azure DevOps Build pipeline "UI Tests".
 
-By following the instructions in this section, you can backup and restore all the data from your local environment, including content and statistics databases and all important Azure storage blobs and table.
+You can run the "UI Tests" pipeline using the branch with your changes, and it will produce an artifact you can download with the test report, screenshots, etc.
 
-You can currently only backup and restore data on your local environment on Windows. This is because currently you can only emulate Azure storage tables on Windows. At the time of writing [Azurite V3 only has support for blobs and queues, not tables](https://github.com/Azure/Azurite).
+If you wish to run tests against other environments, it's recommended to do so from your own machine. Intermittent failures that only happen on the pipeline are rare with non-dev environments, as only the general_public tests are run against them, so for now this has sufficed.
```

Comment:
It is tough to change the "UI Tests" pipeline job to run against other environment, as it will involve updating various pipeline variables and `pipeline-run-rf-tests.sh` to run against the environment you want. So I've changed this section to say if you wish to run tests against other environments, its best done locally from your own machine.